xblock/field_data.py

```diff
@@ -88,7 +91,7 @@ def set_many(self, block, update_dict):
         :param update_dict: A map of field names to their new values
         :type update_dict: dict
         """
-        for key, value in update_dict.items():
+        for key, value in list(update_dict.items()):
```

Comment:
I just noticed all of these, from the autoconversion.  I will update them to use `six.iteritems()` (and related functions) to avoid the extra memory usage.